data/assets/scene/solarsystem/planets/mars/moons/deimos.asset

```diff
@@ -25,10 +33,22 @@ local Deimos = {
     }
   },
   Renderable = {
-    Type = "RenderableGlobe",
-    Radii = { 15000, 12200, 11000 },
-    SegmentsPerPatch = 90,
-    Layers = {}
+    Type = "RenderableModel",
+    GeometryFile = model .. "/Deimos_1_1000.glb",
+    ModelScale = 1000,
```

Comment:
I find it very confusing as well, but after many (probably for Malin frustrating) discussions I think it is necessary to some extent since it covers a different error mode of ("model scale does not agree with OpenSpace scale). 
Mode 1:  Modeller specified all values in centimeters -> ModelScale = "Centimeters"
Mode 2:  Modeller specified all values thinking it was feet but actually writes them out as meters ->   ModelScale = "Feet", InvertModelScale = true
http://wiki.openspaceproject.com/docs/builders/models/model-scale

But yes, let's talk about it in the meeting

Reply:
So yeah it is very confusing but the reason is that different modeling software behave differently and the modellers can also behave differently as well. Most of the time the "Centimeters" unit is enough but for some models it isn't and then the inverted key can be used.